### PR TITLE
freeze d2l lib for v1.0.0 release

### PR DESCRIPTION
**Pull request by AnirudhDagar**
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

File: d2l/__init__.py [hunk 1]
```diff
@@ -9,4 +9,4 @@
 
 """
 
-__version__ = "1.0.0-beta0"
+__version__ = "1.0.0"
```
Comment (astonzhang) [inline d2l/__init__.py:12]: please revert this for now

File: .github/workflow_scripts/utils.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # By default, all builds are cached
-DISABLE_CACHE=false  # Eg. 'true' or 'false'
+DISABLE_CACHE=true  # Eg. 'true' or 'false'
```
Comment (astonzhang) [inline .github/workflow_scripts/utils.sh:4]: remember to change it back :)

Comment (AnirudhDagar) [reply .github/workflow_scripts/utils.sh:4]: yes ofc!

**Comment by astonzhang** [comment]
This is the version on my local machine (evaluated with PyTorch without issues):
py3.9
jupyter                           1.0.0
numpy                             1.24.1
matplotlib                        3.5.2
matplotlib-inline                 0.1.3
requests                          2.28.1
pandas                            1.4.3

File: d2l/__init__.py (hunk 1 above)
Comment (AnirudhDagar) [reply d2l/__init__.py]: done

**Comment by AnirudhDagar** [comment]
> This is the version on my local machine (evaluated with PyTorch without issues):
py3.9
jupyter 1.0.0
numpy 1.24.1
matplotlib 3.5.2
matplotlib-inline 0.1.3
requests 2.28.1
pandas 1.4.3

discussed offline and resolved

File: .github/workflow_scripts/utils.sh (hunk 2 above)
Comment (AnirudhDagar) [reply .github/workflow_scripts/utils.sh]: done

**Pull request merged**